bin/_test-run.sh

```diff
@@ -173,13 +173,19 @@ install_stable() {
     local linkerd_path=$tmp/.linkerd2/bin/linkerd
     local stable_namespace=$1
     local test_app_namespace=$stable_namespace-upgrade-test
-    $linkerd_path install --linkerd-namespace="$stable_namespace" | kubectl --context=$k8s_context apply -f - > /dev/null 2>&1
-    $linkerd_path check --linkerd-namespace="$stable_namespace" > /dev/null 2>&1
+    (
+        set -x
+        $linkerd_path install --linkerd-namespace="$stable_namespace" | kubectl --context=$k8s_context apply -f - 2>&1
+        $linkerd_path check --linkerd-namespace="$stable_namespace" 2>&1
+    )
```

Comment:
I would like to suggest adding double quotes for `$linkerd_path` and `$k8s_context` here now that these lines are being altered in other ways anyway (there will be a merge conflict to solve for https://github.com/linkerd/linkerd2/pull/4406 either way but that's a quick update).

Reply:
Added the quotes, thanks!